e2e/test/helpers/templates/FaultInjection.cs

```diff
@@ -82,7 +82,7 @@ public static async Task ActivateFaultInjectionAsync(IotHubClientTransportSettin
 
             try
             {
-                using var cts = new CancellationTokenSource(delay);
+                using var cts = new CancellationTokenSource(LatencyTimeBuffer);
```

Comment:
In some cases, it is taking over 5secs for the fault injection message to be ack'ed; causing tests to fail with the error "fault was not injected". Increasing the cancellation token validity to 10 secs.